docs/getting_started.md

```diff
@@ -0,0 +1,166 @@
+
+## Quality control  
+
+After we have imported the RDS file into R and accessed the `SingleCellExperiment` object, we can begin working with the data. 
+Before we perform any downstream steps, it is recommended to remove any low quality cells from our dataset.
+This would include cells that may be dying or damaged, showing a higher percentage of reads coming from mitochondrial cells. 
+Low quality cells may also be present due to technical artifacts such as inefficient reverse transcription or PCR amplification resulting in a lower number of total reads and unique genes identified. 
+
+For the filtered `SingleCellExperiment` object present in all `filtered.rds` files, we have used [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html), a data driven approach to predict low-quality cells, to identify cells that should be removed prior to downstream analyses. 
+`miQC` jointly models the proportion of mitochondrial reads and the number of unique genes detected in each cell to calculate the probability of a cell being compromised (i.e. dead or damaged). 
+High-quality cells are those with a low probability of being being compromised (< 0.75) or sufficiently low mitochondrial content. 
+We have already incorporated the `miQC` results into the {ref}`colData() of the SingleCellExperiment object<sce_file_contents:cell metrics>`. 
+All cells that are identified as low-quality cells will have `FALSE` in the `miQC_pass` column of the `colData()`. 
+
+The following commands can be used to remove those cells prior to downstream analysis: 
+
+```r
+# create a vector of barcode names for cells that are not compromised 
+# the `$` notation denotes access to the colData slot of the `SingleCellExperiment` object 
+cells_to_keep <- colnames(sce$miQC_pass == TRUE)
+
+# filter the `SingleCellExperiment`
+filtered_sce <- sce[, cells_to_keep]
+```
+For more information on using `miQC` for filtering cells, see the following resources: 
+- [`miQC` vignette](https://bioconductor.org/packages/release/bioc/vignettes/miQC/inst/doc/miQC.html)
+- [Hippen _et al._ 2021](https://doi.org/10.1371/journal.pcbi.1009290)
+
+Alternatively, you can directly filter cells based on having a minimum number of unique genes, total reads, and a maximum fraction of mitochondrial reads. 
+Using the function, [`scuttle::addPerCellQCMetrics()`](https://rdrr.io/github/LTLA/scuttle/man/addPerCellQCMetrics.html), we can calculate the total RNA counts, unique genes, and mitochondrial fraction for each cell and store the results in the `colData()`. 
+These metrics can be used to directly filter the `SingleCellExperiment` object based on informed minimum thresholds. 
+The `filtered.rds` files contain `SingleCellExperiment` objects with these metrics added to the `colData()` and you can read more about all metrics that are included in the {ref}`cell metrics section of the SingleCellExperiment file contents<sce_file_contents:cell metrics>`.
```

Comment:
So these results are already in the object right? (I am under the impression that you need to add the QC metrics _before_ you run `miQC`.) Start with the fact that these are already present and you can use them to filter.